apps/trading-e2e/src/integration/withdrawals.feature

```diff
@@ -0,0 +1,64 @@
+@ignore
+Feature: Withdrawals
+
+  Scenario: Can prepare a withdrawal
+    Given I am on withdrawals page
+    And I connect eth wallet
+    And I connect vega wallet
```

Comment:
question (non-blocking): should we validate the appropriate UI to prompt users to connect is displayed before each of these?

Reply:
I think so but I think that could exist in the step definition
```
And('I connect vega wallet', () => {
  cy.contains('Plesae connect to vega wallt')
  cy.click(this.connectButton)
  // etc
})
```